boot.py

```diff
@@ -1,3 +1,3 @@
 import supervisor
 
-supervisor.set_next_stack_limit(4096 + 1024)
+supervisor.set_next_stack_limit(4096 + 768)
```

Comment:
This change is 100% required unless we freeze the library in. Any higher and it would not have enough memory to run, and lower and we would have to small of a stack, and also crash. I believe that we should release frozen builds for the supported boards, but in the mean time, this works on every single config I have available to test.